packages/charts/src/vaadin-chart.js

```diff
@@ -487,6 +488,7 @@ class Chart extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))) {
     super();
 
     this._baseConfig = {
+      annotations: [],
```

Comment:
There are no tests attached. Can this feature be tested?

Reply:
I added this part as a precaution. Annotations currently work the same without this line. I can add a test if you think it adds value. Or alternatively, I can remove this line until we actually need the empty array in the default config.

Reply:
I didn't mean this line in particular. But since this PR adds support for annotations, there could be a check that the feature is actually available.

Reply:
That makes perfect sense. I will add a test.

Reply:
I added 2 simple tests with the latest commits.